### ISSUE TITLE
Escape issues with union types on propertiesFormat: "Table"

### ISSUE THREAD

**Issue by adambergman** [tgreyuk/typedoc-plugin-markdown#423]
When setting `"propertiesFormat"` to `"Table"`, it appears that the pipes in union types are being escaped incorrectly and  spaces are inserted between the `\\` and `|` as if they were marking table columns. This was happening in next.8 and continues in next.9.

Block comment + TS Interface
```ts
/**
 * Options interface for `call` method
 */
export interface CallOptions {
  /**
   *  A component or string reference
   */
  component?: Component | string | null;
}
```

Generated markdown
```markdown
#### Properties
| Property  | Modifiers  | Type        | Description                                       |
| :-------- | :--------- | :---------- | :------------------------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| component | `optional` | `null` \\   | `string` \\                                       | `Component` | A component or string reference |
```

**Comment by tgreyuk** [comment]
Thanks - this is super useful feedback. Will fix it now.

**Comment by tgreyuk** [comment]
This should be fixed in `typedoc-plugin-markdown@4.0.0-next.10`
